Modules/Quotas/test/Quotas.Application.Tests/Tests/Repositories/MessagesRepositoryTests.cs

```diff
@@ -14,16 +14,18 @@
 namespace Backbone.Modules.Quotas.Application.Tests.Tests.Repositories;
 public class MessagesRepositoryTests
 {
+    private static readonly DateTime HALF_OF_MONTH = new(DateTime.Now.Year, DateTime.Now.Month, 15);
```

Comment:
Can't we just call this `TODAY`? And then I would also set `SystemTime` to this value in the constructor of the test.

Otherwise, nobody understands why YESTERDAY is set to HALF_OF_MONTH-1

Reply:
of course, that helps!